pyproject.toml

```diff
@@ -110,8 +111,13 @@ filterwarnings = [
   "ignore:There is no current event loop:DeprecationWarning",
   "ignore:DeprecationWarning*.py:DeprecationWarning",
   "ignore:CoverageWarning*.py:",
-  "ignore::DeprecationWarning"
+  "ignore::DeprecationWarning",
 ]
 
 addopts = "--cov=auto_dev --cov-report html"
 
+[tool.coverage.run]
+source = ["."]
+
+[tool.coverage.paths]
+source = ["auto_dev", "/tmp/*/auto_dev"]
```

Comment:
this increased test coverage locally from ~34% to ~53%

We have some issue on CI generating the coverage report tho, it appears (see [CI run](https://github.com/8ball030/auto_dev/actions/runs/14454667612/job/40535081352#step:10:24))
```
1s
Run MishaKav/pytest-coverage-comment@main
File read successfully "/Users/runner/work/auto_dev/auto_dev/./coverage-report.txt"
Error: Coverage file "/Users/runner/work/auto_dev/auto_dev/./coverage-report.txt" has bad format or wrong data
./coverage-report.txt
Nothing to report
```